extend documentation around chargeback statement billing info

sttomm opened the pull request:
- added payment properties

Concerns CU-6yjzzj

meshcloud-gmbh commented:
Task linked: [CU-6yjzzj Extend Chargeback Export to contain more Payment Info](https://app.clickup.com/t/6yjzzj)

JohannesRudolph commented on docs/meshstack.billing.md:
```diff
@@ -239,6 +239,24 @@ The following configuration options are available at `mesh.kraken.productcatalog
 
 Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
 
+The attributes that shall be part of the billing info on the chargeback statements can be configured as relevant meta keys at `mesh.kraken.api.statements`
```
f: This does not look like it's sourced from a proper snippet like 
`<!--snippet:mesh.kraken.api.statements-->`

did you write it yourself @sttomm ?

sttomm replied:
yeah, you are right. i manually added it. I now added it as a real snippet

sttomm commented:
@chewbeccy I extended the config part of this for the operator. can you please check whether the new part in meshstack.billing.md is fine regarding its text?

JohannesRudolph commented:
@sttomm you'll have to regenerate snippets with the latest develop branch to resolve conflicts (snippets now use dhall code blocks)

sttomm commented:
i added the note about payment method and tags being taken at the point in time when the chargeback statement is generated. i also updated documentation around payment methods because it was quite deprecated. And i updated the snippet to explain better how to use payment info and tags.
